src/Components/ExpandedRulesDetails.js/ExpandedRulesDetails.js

```diff
@@ -61,20 +67,24 @@ const ExpandedRulesDetails = ({ more_info, resolution, objects }) => {
           <Table borders={'compactBorderless'}>
```

Comment:
To get rid of this error, specify a aria label
![image](https://github.com/RedHatInsights/ocp-advisor-frontend/assets/20592948/f876d1f9-80bd-4d3a-9cdd-8721a9114069)

```suggestion
          <Table borders={'compactBorderless'} aria-label="Objects table">
```

Reply:
its in!